src/components/helpers.js

```diff
@@ -120,7 +121,7 @@ export const setCookies = (rid, token) => {
 	document.cookie = `rc_room_type=l; path=/; ${ getSecureCookieSettings() }`;
 };
 
-export const createToken = () => Math.random().toString(36).substring(2, 15) + Math.random().toString(36).substring(2, 15);
+export const createToken = () => crypto.randomBytes(128).toString('hex');
```

Comment:
Apart from this, I noticed that there are a couple of places elsewhere in the code where we're using this same logic to generate a token :/ Can we please modify that as well to cover all the cases

![image](https://user-images.githubusercontent.com/34130764/145027705-6038b020-29f2-476c-ac03-1a0791d8f3d6.png)

Reply:
great point Murtaza, doing it now.